docs/_developer_onboarding.md

```diff
@@ -130,9 +130,9 @@ Prerequisites:
   ```
   cargo install cargo-ndk
   ```
-* Install the Android Rust toolchain:
+* Install the Android Rust toolchain for your machine's hardware:
   ```
-  rustup target add aarch64-linux-android
+  rustup target add `uname -m`-linux-android
```

Comment:
On my computer (MacOs) `uname -m` return `arm64` so this is not equal to `aarch64`. Does it work on your side?

Reply:
Oh, bother. Yes, it works here, on an x86_64, but that's not saying much.

I think for these developer onboarding instructions we should `rustup target add` all the potential architectures, but that doesn't help with the `build_rust_sdk.sh` script.

Reply:
ok, fixed, I hope